subprojects/packagefiles/nowide/meson.build

```diff
@@ -20,10 +20,14 @@ subdir('src')  # fills in 'sources'
 
 includes = include_directories('include')
 
+subdir('include/nowide')  # calls install_headers()
```

Comment:
You could actually simplify this with:
```meson
install_subdir('include/nowide', install_dir: get_option('includedir'))
```

Since the entire directory gets installed as-is.

Reply:
TIL ...

Reply:
Since Meson 0.63.0 -- but this is pretty bleeding edge -- you can also do `install_headers('list', 'of', 'headers/in/subdirs', preserve_path: true)` to name each installed file but not have to duplicate relative path information.

But really, install_subdir is fine for common cases like this where there's a well-formed include directory.